src/core/src/op/paged_attention.cpp

```diff
@@ -137,16 +215,27 @@ void PagedAttentionExtension::validate_and_infer_types() {
     } else {
         set_output_type(1, m_output_type[1], {Dimension::dynamic()});
     }
+
+    set_output_type(2, m_output_type[2], {block_ps});
+    set_output_type(3, m_output_type[3], {block_ps});
+    set_output_type(3, m_output_type[4], {block_ps});
```

Comment:
Incorrect index, should be either  
```suggestion
    set_output_type(3, m_output_type[3], {block_ps});
    set_output_type(4, m_output_type[4], {block_ps});
```
or removed:
```suggestion
    set_output_type(3, m_output_type[3], {block_ps});
```